doc/docker/debian/jessie/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM debian:jessie
+
+ENV LANG C.UTF-8
+ENV LANGUAGE C.UTF-8
+ENV LC_ALL C.UTF-8
+
+RUN apt-get update && apt-get -y install \
+    curl \
```

Comment:
Is there a way to move all deps to one place? Ideally fixing #1016 and #1886

Reply:
Yes, but with #1016 we could add an entry for every distribution (+versions of distributions) we want to support.